### PR TITLE
Fixes on custody spec and samples

### PR DESCRIPTION
**Pull request by andiahlm**
Fixes on custody v3.0.1 after resulting from sample validation

### CODE REVIEW

File: CustodyServices/Specs/Resources/Transaction.yaml [hunk 1]
```diff
@@ -60,6 +60,8 @@ properties:
     $ref: 'FinancialInstrument.yaml'
   triggeringQuantity:
     $ref: '../Types/Quantity.yaml'
+  triggeringPrice:
```
Comment (TLusti) [inline CustodyServices/Specs/Resources/Transaction.yaml:63]: The addition of a new attribute should raise the version number of the API

**Review by TLusti — COMMENTED** [review]
The addition of a new attribute should raise the version number of the API, but the API file was not changed

File: CustodyServices/Specs/Resources/Transaction.yaml (hunk 1 above)
Comment (andiahlm) [reply CustodyServices/Specs/Resources/Transaction.yaml:63]: @TLusti This is still open and will be done when we determine the order of PR merges (on the last we will raise the version number)

**Comment by andiahlm** [comment]
Resolves #142 
Branch was rebased

**Comment by andiahlm** [comment]
@TLusti all changes for v3.1.0 are now collected in this branch. The version is set to 3.1.0-draft

**Review by TLusti — APPROVED** [review]
LGTM

**Pull request merged**